frontend/web/src/app/services/digital-print/digital-print.service.ts

```diff
@@ -43,6 +44,10 @@ export class DigitalPrintService {
       });
   }
 
+  downloadMagicsFile(id: number): Observable<MagicsFile> {
+    return this.client.get(this.endpoint + '/' + id);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)